### ISSUE TITLE
Syntax: writing -(1)

### ISSUE THREAD

**Issue by UWN** [infradig/trealla#363]
```
?- writeq(-(1)).
-1   true. % unexpected and incorrect
```

[#135](http://www.complang.tuwien.ac.at/ulrich/iso-prolog/conformity_testing#135).
Since `(-)/1` is a predefined prefix operator (6.3.4.4), it must be used, thus

`- (1)`

is expected. That is, `-(1)` is not conforming, since this may only be used if `-` is not a prefix (nor a postfix) operator.

**Comment by infradig** [comment]
What about this?

~/trealla (devel) $ scryer
?- writeq(-(0)), nl.
- 0
   true.
?-

**Comment by flexoron** [comment]
or this?
```
?- writeq(-(-1)), nl.
- -1
   true.
```

**Comment by flexoron** [comment]
```
Trealla Prolog (c) Infradig 2020-2021, v1.18.40
?- writeq(-(-1- +1)), nl.
- (-1-(+1))
   true.
```
```
$ scryer-prolog
?- writeq(-(-1- +1)), nl.
- (-1- +1)
   true.
```
both are correct?

**Comment by UWN** [comment]
> or this?
> 
> ```
> ?- writeq(-(-1)), nl.
> - -1
>    true.
> 
> ?- writeq(-(-1-1)), nl.
> - (-1-1)   
>    true.
> ```

Both are conforming. The first is [#182](http://www.complang.tuwien.ac.at/ulrich/iso-prolog/conformity_testing#182).

**Comment by UWN** [comment]
> both are correct?

To clarify: conforming means conforming to the standard, and here [7.10.5](http://www.complang.tuwien.ac.at/ulrich/iso-prolog/draft-7.10.5).

Incorrect, often means, cannot be read back with the same operators to the same term.

**Comment by UWN** [comment]
Closing, original issue is solved, issues mentioned by flexoron are in #368
